app/src/test/java/com/courseproject/tindar/usecases/login/LoginInteractorUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package com.courseproject.tindar.usecases.login;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+public class LoginInteractorUnitTest {
+    /** This class tests the implementation and return values of LoginInteractor **/
+    private static final String email = "bell@exampleemail.com";
+    private static final String password = "somepassword";
+    private static final String userId = "1";
+
+    private static class MockLoginDsGateway implements LoginDsGateway {
+        private static final String email = "bell@exampleemail.com";
+        private static final String password = "somepassword";
+        private static final String userId = "1";
+
+        public MockLoginDsGateway(){}
```

Comment:
you probably don't have to write an empty constructor. Should be just given as default.